### PR TITLE
Add yolov8l.onnx to samples

### PR DESCRIPTION
**Pull request by richard28039**
### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under Apache 2 License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or another license that is incompatible with OpenCV
- [x] The PR is proposed to the proper branch
- [ ] There is a reference to the original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [ ] The feature is well documented and sample code can be built with the project CMake

Hello, I noticed that the /samples/dnn/models.yml said it should be used for all yolov8 models, but the YOLOv8l is not included in the file, so I added it to the file, thanks.
image

### CODE REVIEW

**Comment by asmorkalov** [comment]
We already have several yolov8 models in samples. We do not need yet another one, if it just brings fatter backbone, but not some DNN features.

**Comment by Tsai-chia-hsiang** [comment]
But we think the model.yml file.

https://github.com/opencv/opencv/blob/24907f35a32b75d9c5b34a203b44327ba53f86f4/samples/dnn/models.yml#L22

It says that it supports the YOLOv8L model, but there is no link for the YOLOv8L model.
Thus, it's better to support YOLOv8L to satisfy what Line 22 says. 

Thanks!

File: samples/dnn/models.yml [hunk 1]
```diff
@@ -76,6 +76,20 @@ yolov8m:
   background_label_id: 0
   sample: "yolo_detector"
 
+yolov8l:
```
Comment (asmorkalov) [inline samples/dnn/models.yml:79]:
```
opencv-master/samples/dnn$ python3 ./download_models.py yolov8l
DownloadInstance <opencv_face_detector_fp16>
DownloadInstance <opencv_face_detector_uint8>
DownloadInstance <opencv_fd>
DownloadInstance <yolov8x>
DownloadInstance <yolov8s>
DownloadInstance <yolov8n>
DownloadInstance <yolov8m>
DownloadInstance <yolov8l>
Model: yolov8l
  Working on yolov8l
  Getting file yolov8l.onnx
  expected SHA1: 5c5cc8ad7ab50973a84444bae4e613065a479ea1
  there is already a file with the same name
  actual SHA1:462df53ca3a85d110bf6be7fc2e2bb1277124395
  hash didn't match, loading file
  hash check failed - loading
  Preparing to download file yolov8l.onnx
  expected SHA1: 5c5cc8ad7ab50973a84444bae4e613065a479ea1
  there is already a file with the same name
  actual SHA1:462df53ca3a85d110bf6be7fc2e2bb1277124395
  200 OK [166.78626346588135 Mb]
  progress >>>>>>>>>>>>>>>>>  Downloaded yolov8l.onnx with size 166.78626346588135 Mb
  expected SHA1: 5c5cc8ad7ab50973a84444bae4e613065a479ea1
  there is already a file with the same name
  actual SHA1:462df53ca3a85d110bf6be7fc2e2bb1277124395
  There was some problem with loading file yolov8l.onnx for yolov8l
  Exception: Hash mismatch: expected 5c5cc8ad7ab50973a84444bae4e613065a479ea1 vs actual of 462df53ca3a85d110bf6be7fc2e2bb1277124395
```

**Review by asmorkalov — CHANGES_REQUESTED** [review]

**Comment by asmorkalov** [comment]
Looks like the model was exported with another parameters than all other yolos we have in the sample. The sample generate a lot of garbage boxes that do not correspond to real objects. Command for the sample: `./bin/example_dnn_yolo_detector --model=./yolov8l.onnx --yolo=yolov8`

**Comment by richard28039** [comment]
Hi, the ```mode.yml``` scale is 0.0039, and it's necessary to give this argument.

If not passing --scale = 0.0039:
```
./yolo_detecor --input=./261902.jpg --yolo=yolov8 --model=./yolov8l.onnx 
```
The result of yolo_detecor.cpp is

testyolov8l_no_scale

After passing --scale = 0.0039
```
./yolo_detecor --input=./261902.jpg --yolo=yolov8 --model=./yolov8l.onnx  --scale=0.0039 
```
The result becomes:

testyolov8l_with_scale

Please take a look, Thanks!

**Comment by richard28039** [comment]
Hello, I found some mistakes when testing, so I removed the unnecessary whitespace in line92 and pushed the new commit; thanks.
```
Samples/dnn/models.yml:92: trailing whitespace.
```

**Review by asmorkalov — APPROVED** [review]

**Pull request merged**